hack/update-kubernetes-library.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
```

Comment:
Where is this script used?

Reply:
I use this script for generating the `replace k8s.io/*` directive in `go.mod`.

It will only be used when updating kubernetes library.